source/learning-wazuh/audit-commands.rst

```diff
@@ -406,23 +406,10 @@ Observe the order in which our child rules are evaluated
 #. Search Kibana for ``data.audit.command:sleep`` to find the resulting event.
    Copy the ``full_log`` value.
 
-#. Run ``/var/ossec/bin/wazuh-logtest -v`` on the Wazuh Manager and paste in
+#. Run ``/var/ossec/bin/wazuh-logtest`` on the Wazuh Manager and paste in
    the ``full_log`` value from above.
 
-#. Carefully note the order in which child rules of "80792 - Audit: Command"
-   were evaluated:
-
-    .. code-block:: none
-       :class: output
-
-       ...
-       Trying rule: 80789 - Audit: Watch - Execute access: $(audit.file.name)
-       Trying rule: 80792 - Audit: Command: $(audit.exe)
-          *Rule 80792 matched.
-          *Trying child rules.
-       Trying rule: 100220 - Ignore pings of 8.8.8.8
-       Trying rule: 100210 - Audit: Highly Suspicious Command: $(audit.exe)
-       Trying rule: 100200 - Audit: Suspicious Command: $(audit.exe)
+#. Note the rule 100200 was evaluated.
```

Comment:
Please change this sentence since rule 100200 is fired but a lot of alerts before are also evaluated:

```
#. Verify rule 100200 matches.